conbench/templates/benchmark-entity.html

```diff
@@ -104,6 +104,17 @@
               {% endif %}
             </li>
           {% endfor %}
+          {% if benchmark.optional_info %}
+            <li class="list-group-item active">Additional information</li>
```

Comment:
This section needs to be called something else.

We already have `Additional Information` for benchmark.info.